src/components/LessonListsByStates/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import {
+  newLessonsActionText,
+  inProgressTitleText,
+  inQueueTitleText,
+  publishedTitleText,
+  noInProgressLessonsDescriptionText,
+  noInQueueLessonsDescriptionText,
+  noPublishedLessonsDescriptionText,
+} from 'utils/text'
+import {inProgressLessonStates, publishedLessonStates} from 'utils/lessonStates'
+import LessonList from 'components/LessonList'
+import Tabs from 'components/Tabs'
+import Prompt from 'components/Prompt'
+
+export default ({instructor}) => (
+  <Tabs groups={[
+    {
+      title: inProgressTitleText,
+      component: (
+        <LessonList
+          states={inProgressLessonStates}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noInProgressLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+    {
+      title: inQueueTitleText,
+      component: (
+        <LessonList
+          states={['approved']}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noInQueueLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+    {
+      title: publishedTitleText,
+      component: (
+        <LessonList
+          states={publishedLessonStates}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noPublishedLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+  ]} />
+)
```

Comment:
@joelhooks These are the `Button` component implemented in `egghead-ui`; there is a `small` option which the white "View" buttons already are; for those to be smaller the style guide would need to be updated. The `Create a new lesson` blue button is the default size so it could have the `small` attribute to match the size of the white buttons; but because this button is the main call to action, I don't think the `small` option semantically makes sense here, or all buttons would be `small`; if the buttons seem to big we could see if the style guide could be updated and pull down from there: https://app.zeplin.io/project.html#pid=5821fdf6bfe6e810074eace4&sid=582375cf0a4d883e6730d490